Assets/MixedRealityToolkit/Definitions/Utilities/TransformationSpaceType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    public enum TransformationSpaceType
+    {
+        /// <summary>
+        /// Use the define position, rotation or scale in local space
+        /// </summary>
+        LocalSpace = 0,
+
+        /// <summary>
+        /// Use the define position, rotation or scale in world space
+        /// </summary>
+        WorldSpace
```

Comment:
Hi @Alexees , I'm curious....did you ever have a case when you wanted the offset to be in world space? It seems by default the offset is in local space alway.

Reply:
I did not think about use cases really :)
I was just a little frustrated about Orbital that I thought, make it as flexible as it can be. If you can follow something, with offset, and at the same time look at something, with offset. Maybe someone might find this useful. 
> It seems by default the offset is in local space always

I don't understand, some value has to be the default. It certainly doesn't mean worldspace isn't used.